source/core/GenericFunctions.cpp

```diff
@@ -1290,6 +1290,17 @@ VIREO_FUNCTION_SIGNATURET(VectorOpInternal, VectorOpInstruction)
                 *(Double *)scalarOpSnippet->_p2 = isIdentityOne;
             break;
         }
+        case kEncoding_Cluster:
+            if (array->ElementType()->IsComplex()) {
+                if (elementSize == 8) {
```

Comment:
I think it should be sizeof(ComplexSingle) but I didn't do this because the type is not exposed in this file; it's local to Math.cpp where all of the actual complex support is and I didn't want to move it to a header file.  However, sizeof(float)*2 is an acceptable compromise.